File: src/main/java/ru/arc/invest/BusinessCore.java
package ru.arc.invest;

import ru.arc.invest.upgrades.Upgrade;

import java.util.List;

public class BusinessCore {

    List<Upgrade> upgrades;
    long productionCycleMs;
    double failureChance;
    double speedup;
    long lastCycleTimestamp;
    Production production;

}

File: src/main/kotlin/ru/arc/invest/BusinessCore.kt
package ru.arc.invest

import ru.arc.invest.upgrades.Upgrade

class BusinessCore {
    var upgrades: List<Upgrade>? = null
    var productionCycleMs: Long = 0
    var failureChance: Double = 0.0
    var speedup: Double = 0.0
    var lastCycleTimestamp: Long = 0
    var production: Production? = null
}
